src/server/contest/views.py

```diff
@@ -2,6 +2,8 @@
 from time import mktime
 from shutil import copyfile
 from math import isclose
+from os import listdir
```

Comment:
`pathlib` is recommended instead of `os` module.

Reply:
Agreed. Since me recently adapted `pathlib`, let's continue using it for the sake of consistency, if nothing else.